### PR TITLE
feat: frame mod

### PR DESCRIPTION
**Pull request by davidfurlong**
## Change Summary

Adds a Farcaster Frames rendering Mod

## Merge Checklist

- [x] Have a sample implementation of `SENDFCFRAMEACTION`, waiting on Neynar
- [x] Update example code
- [x] testing

- [x] PR has a changeset
- [x] PR includes documentation if necessary
- [x] PR updates the [rich-embed examples](https://github.com/mod-protocol/mod/blob/main/examples/nextjs-shadcn/src/app/embeds.tsx) if necessary
- [x] includes a parallel PR for [Mod-starter](https://github.com/mod-protocol/mod-starter) and the gateway if necessary

### CODE REVIEW

File: examples/nextjs-shadcn/src/app/signer/route.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+if (typeof process.env.FARCASTER_DEVELOPER_FID === "undefined") {
+  throw new Error("FARCASTER_DEVELOPER_FID is not defined");
+}
+const FARCASTER_DEVELOPER_FID = process.env.FARCASTER_DEVELOPER_FID;
+
+export async function GET(req: NextRequest, res: NextResponse) {
+  // Do whatever you want
+  try {
+    const response = await axios.get(
```
Comment (stephancill) [inline examples/nextjs-shadcn/src/app/signer/route.ts:18]: is there a reason why axios is used over fetch here?

File: packages/farcaster/src/frame-actions.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
+// import {
```
Comment (stephancill) [inline packages/farcaster/src/frame-actions.ts:1]: empty file

**Review by stephancill — APPROVED** [review]

File: examples/nextjs-shadcn/src/app/signer/route.ts (hunk 1 above)
Comment (davidfurlong) [reply examples/nextjs-shadcn/src/app/signer/route.ts:18]: copy pasta from neynar signer example - was tempted to rewrite but couldn't be bothered

File: packages/farcaster/src/frame-actions.ts (hunk 2 above)
Comment (davidfurlong) [reply packages/farcaster/src/frame-actions.ts:1]: oops

**Pull request merged**